src/confluent/azext_confluent/manual/commands.py

```diff
@@ -35,3 +35,6 @@ def load_command_table(self, _):
     with self.command_group('confluent terms', confluent_marketplace_agreement,
                             client_factory=cf_marketplace_agreement, deprecate_info=g.deprecate(redirect='az term', hide=True)) as g:
         g.custom_command('list', 'confluent_terms_list')
+    
+    with self.command_group('confluent', is_experimental=False):
```

Comment:
No need to add this as by default `is_experimental` is false.

Reply:
Ok, I have made the changes and have pushed the same !